string text
string speakerName
---
string result
string fail_reason
